### DISCUSSION TITLE
Is there no way to import a new instance by exporting the original instance data?

### DISCUSSION THREAD

**Discussion by wqwwffsytxwp** [firefly-iii/firefly-iii#9000] [Firefly III]
Is there no way to import a new instance by exporting the original instance data?

**Comment by wqwwffsytxwp** [comment]
In addition, after a long period of time, after the transaction reaches 6K or more, the overall operation and the response time of new transactions will become slow.

**Comment by JC5** [comment]
> Is there no way to import a new instance by exporting the original instance data? 

Any database can do export and import :)
